Solution1/BusinessLayer/Services/WeatherService.cs

```diff
@@ -32,7 +32,8 @@ public async Task<WeatherDTO> GetByCityNameAsync(string cityName, CancellationTo
             var validationResult = await _validator
                                             .ValidateAsync(
                                                 new ForecastWeatherRequestDTO() { CityName = cityName },
-                                                options => options.IncludeRuleSets("CityName"));
+                                                options => options.IncludeRuleSets("CityName"),
```

Comment:
every string literal should be constant!!!! what if i will say rename ruleset in the validator, you will go to all services and change the name? what if you will forget to change somewhere?

Reply:
+